### ISSUE TITLE
If system disk is SSD and backups are large, Duplicati's default options will cause write-wear

### ISSUE THREAD

**Issue by kenkendk** [duplicati/duplicati#1032]
_From [rickyorb...@gmail.com](https://code.google.com/u/117733090307531583059/) on June 13, 2014 06:27:00_

As per the title, it may be wise to prompt a first-time user to choose their cache location or provide alternative options.

_Original issue: http://code.google.com/p/duplicati/issues/detail?id=1005_

**Comment by yasinCelik1** [comment]
Is this issue still open?
I only have a SSD as system disk. Is there no option to not use a tempdir on disk? RAM memory is nowadays sufficient enough to hold a couple of hundreds of MB's...

**Comment by kenkendk** [comment]
Yes, the issue is still open. You can create a memory mapped filesystem/drive and set that as the temporary folder.

SSD drives have gotten much better at dealing with write-tear in the meantime though.

**Comment by kaovilai** [comment]
> 
> 
> Yes, the issue is still open. You can create a memory mapped filesystem/drive and set that as the temporary folder.
> 
> SSD drives have gotten much better at dealing with write-tear in the meantime though.

I would love to see this resolved. Maybe a bounty?
On windows I don't think there's a native way to map a ramdisk without external application?
